templates/user/user_disable_enable.html

```diff
@@ -0,0 +1,303 @@
+                  padding: 5px 15px 30px 15px;
+                  border-radius: 4px 4px 0px 0px;
+                  color: #111111;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 48px;
+                  font-weight: 400;
+                  letter-spacing: 4px;
+                  line-height: 48px;
+                "
+              >
+              
+
+                {%if status == "enabled" %}
+                <div class = "cartoons">
+                  <img
+                  src="https://raw.githubusercontent.com/crane-cloud/frontend/develop/public/favicon.png"
+                  class = "item"
+                  height = "100"
+                  width = "150"
+              />
+                <img
+                    src="https://mulapp.s3.eu-west-2.amazonaws.com/IMS/CC_alert.png"
+                    class = "item"
+                    height = "50"
+                    width = "100"
+                />
+              </div>
+
+                {%else%}
+                <div class = "cartoons">
+                  <img
+                  src="https://raw.githubusercontent.com/crane-cloud/frontend/develop/public/favicon.png"
+                  class = "item"
+                  height = "100"
+                  width = "150"
+                />
+                <img
+                    src="https://mulapp.s3.eu-west-2.amazonaws.com/IMS/istockphoto-854560586-170667a.png"
+                    class = "item"
+                    height = "25"
+                    width = "75"
+                />
+                </div>
+                {%endif%}
+                
+              </td>
+            </tr>
+          </table>
+        </td>
+      </tr>
+
+      <tr>
+        <td bgcolor="#f4f4f4" align="center" style="padding: 0px 10px 0px 10px">
+          <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px">
+            <tr>
+              <td
+                bgcolor="#ffffff"
+                align="left"
+                style="
+                  padding: 5px 15px 30px 15px;
+                  color: #666666;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 18px;
+                  font-weight: 400;
+                  line-height: 25px;
+                "
+              >
+                <p style="margin: 0">Hi {{name}},</p>
+              </td>
+            </tr>
+            <tr>
+              <td
+                bgcolor="#ffffff"
+                align="left"
+                style="
+                  padding: 5px 15px 30px 15px;
+                  color: #666666;
+                  font-family: 'Roboto', Helvetica, Arial, sans-serif;
+                  font-size: 18px;
+                  font-weight: 400;
+                  line-height: 25px;
+                "
+              >
+                <p style="margin: 0">This is to inform you that your account has been {{status}}. </p>
```

Comment:
Small suggestion but you can improve on the wording much more
```suggestion
                <p style="margin: 0">We regret to inform you that account associated with the address [User's Email]  has been disabled </p>
```